swupd-extract/main.go

```diff
@@ -0,0 +1,516 @@
+package main
+
+// TODO: Flag to ignore included bundles (only extract the requested). Still need to parse
```

Comment:
While this ~might~ have some uses, I think you should always add the includes if you're attempting to create a full, usable bundle, or it won't work just installing editors for example. Includes are a space saving abstraction, so I think of includes more as Requires(thing) which reduces our bundle def size and manifest sizes. I would possibly be ok if they were providing the other deps themselves, is that the case this would cover? 
And for skipping packs, probably OK to do.

Reply:
> you should always add the includes if you're attempting to create a full, usable bundle

That is the current behavior of the program :)

> I would possibly be ok if they were providing the other deps themselves, is that the case this would cover?

Not really what I was thinking. This program is about extracting content from an existing swupd update repo, it felt like a natural extension if you are using it to perform some kind of validation or inspection or the update repo.

The mindset was more: if you have unzip program, you want to add a feature to just extract a single file out of it.

I didn't have a real use case for this, so just added a note there.

Reply:
Ah ok, thanks for clarifying! I can see that being a useful feature for checking manifest/pack/etc correctness.